src/components/DatePickerRange.react.js

```diff
@@ -1,3 +1,5 @@
+import 'react-dates/initialize'; // https://github.com/airbnb/react-dates/issues/750#issuecomment-335013909
+import 'react-dates/lib/css/_datepicker.css'; // react-dates css
```

Comment:
The CSS loaders will inject the CSS into the head with JS, is that right? I wonder if this will make it harder to override these stylesheets (i.e. will user-added stylesheets get added before or after the injected stylesheets?)

Reply:
> The CSS loaders will inject the CSS into the head with JS, is that right?

Yes, it makes `<style>` tags in the head, which will show up after the stylesheets. 
Should I instead **remove this, plus the `css-loader` plugin for Webpack, and insert the newer react-dates CSS stylesheet**? I hadn't realized that this was happening for serving CSS, as you note below:
> Er, looks like we would only need to remove react-dates. However, we will need to update the stylesheets for the rest of the components that had their versions bumped. We'll also need to include those new stylesheets in the dash_core_components/ folder.

Reply:
Yeah, let's keep the traditional stylesheets for now, that way it's easier for users to override them without removing them

Reply:
Will do.